spring-cloud-netflix-eureka-server/src/main/java/org/springframework/cloud/netflix/eureka/server/InstanceRegistry.java

```diff
@@ -124,6 +121,17 @@ private void handleRegistration(InstanceInfo info, int leaseDuration, boolean is
 		publishEvent(new EurekaInstanceRegisteredEvent(this, info, leaseDuration, isReplication));
 	}
 
+	private void handleRenewal(final String appName, final String serverId, boolean isReplication) {
+		log("renew " + appName + " serverId " + serverId + ", isReplication {}" + isReplication);
```

Comment:
The logging should happen before renewing.

Reply:
Is there a reason why renew is different?

Or should cancellation and registration be changed in that way? 🤔

Reply:
No, sorry. I've misread; Please changed `"renew "` to "renewed "`, so that the sequence of events is clear.

Reply:
fixed :)